src/storage/core.rs

```diff
@@ -905,13 +905,13 @@ where
             .collect::<FuturesUnordered<_>>();
         let total = entries_closed_blobs
             .filter_map(|result| match result {
-                Ok(count) => count,
+                Ok(count) => Some(count),
```

Comment:
If you reverted other changes to this function, then this one should be reverted too.
Also the question: do you plan to start using new functions in a subsequent PR?

Reply:
Which functions?

Reply:
I mean `push_deletion_record` function